### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by hollasch**
Random tweaks

### CODE REVIEW

**Comment by trevordblack** [comment]
Near as I can tell, The changelog needs to have the changes made from surface texture and possibly rotate_y

**Comment by hollasch** [comment]
Regarding the changelog, see the single item under `The Next Week`. For `rotate_y`, are you talking about the bug I just fixed? That was introduced during `dev-major` development, so wouldn't get an entry in the changelog.

**Comment by trevordblack** [comment]
Sounds good.

File: books/RayTracingInOneWeekend.html [hunk 1]
```diff
@@ -1842,7 +1842,8 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
     if (world.hit(r, interval(0.001, infinity), rec)) {
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    [Listing [reflect-tolerance]: <kbd>[main.cc]</kbd> Calculating reflected ray origins with tolerance]
+    [Listing [reflect-tolerance]: <kbd>[main.cc]</kbd>
+    Calculating reflected ray origins with tolerance]
```
Comment (trevordblack) [inline books/RayTracingInOneWeekend.html:1846]: Can/should this be indented?

**Review by trevordblack — APPROVED** [review]
Good change.

These might seem like a bunch of small changes, but they really help remove a lot of the clutter in the source.

File: books/RayTracingInOneWeekend.html (hunk 1 above)
Comment (hollasch) [reply books/RayTracingInOneWeekend.html:1846]: Yeah, I tried it, but honestly it all looks awkward. It's sensitive to which characters the line breaks on, and the multiple different brackets just make it look even more awkward. Flush left was the best I found.

**Pull request merged**